GameServer/ServerContext.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace LeagueSandbox.GameServer
+{
+    class ServerContext
+    {
+        private string _executingDirectory;
```

Comment:
Our guideline calls for `public string ExecutingDirectory { get; private set; }` here